packages/@react-aria/list/docs/useList.mdx

```diff
@@ -0,0 +1,420 @@
+## Example
+
+Lists are [collection components](../react-stately/collections.html) that include rows as child elements.
+In this example, we'll use the standard HTML unordered list elements along with hooks from React
+Aria for each child. You may also use other elements like `<div>` to render these components as appropriate.
+We'll walk through creating the list container and list item, then add some additional behavior such as selection.
+
+The <TypeLink links={docs.links} type={docs.exports.useList} /> hook will be used to render the outer most list element. It uses
+the <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useListState} /> hook to construct the list's collection of rows,
+and manage state such as the focused row and row selection. We'll use the collection to iterate through
+the rows of the List and render the relevant components, which we'll define below.
+
+You may notice the extra `<div>` with `gridCellProps` in our example. This is needed because we are following the [ARIA grid pattern](https://www.w3.org/WAI/ARIA/apg/patterns/grid/).
+
+You may also notice that we included an info button in the rows in our example. This is to highlight `useList`'s ability to handle interactive elements inside a row.
+This differs from [useListBox](useListBox.html), which is a selectable list with no support for interactive elements. If your list will not contain interactive elements,
+[useListBox](useListBox.html) may be a better choice due to better screen reader experience, especially on mobile devices.
+
+```tsx example export=true
+import {useListState} from '@react-stately/list';
+import {mergeProps} from '@react-aria/utils';
+import {useRef} from 'react';
+import {useFocusRing} from '@react-aria/focus';
+import {useButton} from '@react-aria/button';
+import {useList, useListItem} from '@react-aria/list';
+import { Item } from "@react-stately/collections";
+
+function List(props) {
+  let state = useListState(props);
+  let ref = useRef();
+  let { collection } = state;
+  let { gridProps } = useList(props, state, ref);
+
+  return (
+    <ul
+      {...gridProps}
+      ref={ref}
+      style={{
+        padding: 0,
+        margin: '5px 0',
+        listStyle: 'none',
+        border: '1px solid gray',
+        maxWidth: 400
+      }}
+    >
+      {[...collection].map((item) => (
+        <ListItem key={item.key} item={item} state={state} />
+      ))}
+    </ul>
+  );
+}
+
+function ListItem({ item, state }) {
+  let ref = React.useRef();
+
+  let {
+    rowProps,
+    gridCellProps,
+    isPressed,
+    isSelected,
+    isFocused,
+    isDisabled
+  } = useListItem(
+    {
+      node: item
+    },
+    state,
+    ref
+  );
+  let {isFocusVisible, focusProps} = useFocusRing();
+  let showCheckbox = state.selectionManager.selectionMode !== 'none' && state.selectionManager.selectionBehavior === 'toggle';
+  let buttonRef = React.useRef();
+  let { buttonProps } = useButton({
+    onPress: () => alert('Info pressed...')
```

Comment:
Might be nice to add the key/something in the alert message to distinguish between the info buttons presses

Reply:
Went ahead and included a collapsed Button component for this